documentation/resource_apache2_mod.md

```diff
@@ -4,11 +4,13 @@ Sets up configuration file for an Apache module from a template. The template sh
 
 This will use a template resource to write the module's configuration file in the `mods-available` under the Apache configuration directory (`apache_dir`). This is a platform-dependent location.
 
-### Parameters
--   `name` - Type: String - Name of the template.
--   `root_group` - Type: String - Set to override the platforms default root group for the template file.'
+## Properties
+
+- `name` - Type: String - Name of the template.
```

Comment:
We don't want people directly setting name properties so they shouldn't be documented. If there's a need to override it there needs to be a name property